crates/burn-wgpu/src/runtime.rs

```diff
@@ -51,16 +51,52 @@ impl<G: GraphicsApi, F: FloatElement, I: IntElement> Runtime for WgpuRuntime<G,
     }
 }
 
+/// The values that control how a WGPU Runtime will perform its calculations.
+pub struct RuntimeOptions {
+    /// How the buffers are deallocated.
+    pub dealloc_strategy: DeallocStrategy,
+    /// Control the slicing strategy.
+    pub slice_strategy: SliceStrategy,
+    /// Control the amount of compute tasks to be aggregated into a single GPU command.
+    pub max_tasks: usize,
+}
+
+impl Default for RuntimeOptions {
+    fn default() -> Self {
+        let max_tasks = match std::env::var("BURN_WGPU_MAX_TASKS") {
```

Comment:
Can we build conditionally initialize this environment variable depending it's wasm or not?

Reply:
no_std isn't supported for wgpu, but this will work with wasm.

Reply:
Got it. Feels quick and dirty though =D

I'll approve it for now.

Reply:
This option might even go away in future releases